src/net.h

```diff
@@ -359,15 +360,13 @@ class CNode
 
 private:
 
-	CNode(const CNode&);
+    CNode(const CNode&);
     void operator=(const CNode&);
 
-	// Network usage totals
-	 static CCriticalSection cs_totalBytesRecv;
-     static CCriticalSection cs_totalBytesSent;
-     static uint64_t nTotalBytesRecv;
-     static uint64_t nTotalBytesSent;
- 
+    // Network usage totals
+    static std::atomic<uint64_t> nTotalBytesRecv;
+    static std::atomic<uint64_t> nTotalBytesSent;
```

Comment:
While it is static and only instantiated once, I agree that there's no reason for it to be a part of CNode. It can just as well be accessed via loose functions. I'll fix that.